### PR TITLE
Effy update home + global font sizes

### PR DESCRIPTION
**Pull request by effyyzhang**
image
CleanShot 2024-10-16 at 20 22 53@2x

Notebook before and after:
CleanShot 2024-10-17 at 10 44 25@2x
CleanShot 2024-10-16 at 20 28 55@2x

### CODE REVIEW

**Comment by effyyzhang** [comment]
CleanShot 2024-10-16 at 20 41 13@2x

File: packages/components/src/components/ui/dialog.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const DialogDescription = React.forwardRef<
 >(({ className, ...props }, ref) => (
   <DialogPrimitive.Description
     ref={ref}
-    className={cn('text-sm text-tertiary-foreground', className)}
+    className={cn('text-sm font text-tertiary-foreground', className)}
```
Comment (nichochar) [inline packages/components/src/components/ui/dialog.tsx:87]: I dont think this does anything

File: packages/web/src/index.css [hunk 2]
```diff
@@ -154,41 +154,47 @@
 
 @layer components {
   .title {
-    font-size: 3rem;
+    font-size: 2rem;
```
Comment (nichochar) [inline packages/web/src/index.css:157]: Not super critical given that we want to focus on apps, but do these affect the notebook look?

**Review by nichochar — COMMENTED** [review]
One `font` I think we should remove, and maybe a screenshot of the old notebook to make sure things don't look broken, but otherwise this LGTM!

File: packages/web/src/index.css (hunk 2 above)
Comment (effyyzhang) [reply packages/web/src/index.css:157]: This changes the title size of the notebook, no impact on the app. I changed this cause the rest of the font are smaller now.

**Review by nichochar — APPROVED** [review]
LGTM

**Pull request merged**